src/Be.Vlaanderen.Basisregisters.Projector/Controllers/DefaultProjectorController.cs

```diff
@@ -1,20 +1,76 @@
 namespace Be.Vlaanderen.Basisregisters.Projector.Controllers
 {
+    using System.Collections.Generic;
+    using System.Linq;
     using System.Threading;
     using System.Threading.Tasks;
     using ConnectedProjections;
+    using Dapper;
     using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Data.SqlClient;
 
     public abstract partial class DefaultProjectorController : ControllerBase
     {
         private readonly IConnectedProjectionsManager _projectionManager;
+        private readonly string _eventsSchema;
+        private readonly string _eventsConnectionString;
+        private readonly string _baseUri;
 
-        protected DefaultProjectorController(IConnectedProjectionsManager connectedProjectionsManager)
-            => _projectionManager = connectedProjectionsManager;
+        protected DefaultProjectorController(
+            IConnectedProjectionsManager connectedProjectionsManager,
+            string eventsSchema,
+            string eventsConnectionString,
+            string baseUri)
+        {
+            _projectionManager = connectedProjectionsManager;
+            _eventsSchema = eventsSchema;
+            _eventsConnectionString = eventsConnectionString;
+            _baseUri = baseUri;
+        }
 
         [HttpGet]
-        public IActionResult Get()
-            => Ok(_projectionManager.GetRegisteredProjections());
+        public async Task<IActionResult> Get(
+            CancellationToken cancellationToken)
+        {
+            var registeredConnectedProjections = _projectionManager
+                .GetRegisteredProjections()
+                .ToList();
+
+            var responses = await CreateProjectionResponses(registeredConnectedProjections, cancellationToken);
+            var streamPosition = await GetStreamPosition(cancellationToken);
+
+            return Ok(new ProjectionResponseList(responses, _baseUri)
```

Comment:
Don't forget the modify the get projection state in the Public api to accept the new response type